src/Makefile.leveldb.include

```diff
@@ -30,7 +30,7 @@ LEVELDB_CPPFLAGS_INT += -DLEVELDB_PLATFORM_POSIX
 endif
 
 leveldb_libleveldb_a_CPPFLAGS = $(AM_CPPFLAGS) $(LEVELDB_CPPFLAGS_INT) $(LEVELDB_CPPFLAGS)
-leveldb_libleveldb_a_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
+leveldb_libleveldb_a_CXXFLAGS = -Wsuggest-override -Werror=suggest-override $(AM_CXXFLAGS) $(PIE_FLAGS)
```

Comment:
should backport 18843 first

Reply:
Backported #18843 and then some more, resolved!